irohad/ametsuchi/impl/postgres_query_executor.cpp

```diff
@@ -251,9 +253,10 @@ namespace iroha {
                    pending_txs_storage_,
                    std::move(converter),
                    response_factory,
-                   perm_converter),
+                   perm_converter,
+                   log_manager->getChild("Visitor")->getLogger()),
```

Comment:
I guess we should pass parameters for objects which we create via ctor. So better to pass a factory which returns parameters required for child logger instantiation.

Also, I am ok with leaving only todo now.